diff --git a/node_modules/tweek-client/dist/TweekManagementClient/TweekManagementClient.js b/node_modules/tweek-client/dist/TweekManagementClient/TweekManagementClient.js
index 5c74098..587c541 100644
--- a/node_modules/tweek-client/dist/TweekManagementClient/TweekManagementClient.js
+++ b/node_modules/tweek-client/dist/TweekManagementClient/TweekManagementClient.js
@@ -77,10 +77,12 @@ var TweekManagementClient = (function () {
         };
         return this._fetch(url, config).then(noop);
     };
-    TweekManagementClient.prototype.getSuggestions = function (query, count) {
+    TweekManagementClient.prototype.getSuggestions = function (query, options) {
+        options = (typeof(options) === "number" ? {count:options, type:'field'} : options) || {};
         var queryParamsObject = {
             q: query,
-            count: count || defaultSearchCount,
+            count: options.count || defaultSearchCount,
+            type: options.type || 'field',
         };
         var queryString = utils_1.toQueryString(queryParamsObject);
         var url = this.config.baseServiceUrl + "/api/v2/suggestions" + queryString;
diff --git a/node_modules/tweek-client/src/TweekManagementClient/TweekManagementClient.ts b/node_modules/tweek-client/src/TweekManagementClient/TweekManagementClient.ts
index 29686bf..ae938f8 100644
--- a/node_modules/tweek-client/src/TweekManagementClient/TweekManagementClient.ts
+++ b/node_modules/tweek-client/src/TweekManagementClient/TweekManagementClient.ts
@@ -105,10 +105,12 @@ export default class TweekManagementClient implements ITweekManagementClient {
     return this._fetch(url).then(toJson);
   }
 
-  search(query: string, count?: number): Promise<string[]> {
+  search(query: string, options: number | {count?:number, type?: 'free' | 'field'} = {}): Promise<string[]> {
+    options = typeof(options) === "number" ? {count:options, type:'field'} : options;
     const queryParamsObject = {
       q: query,
-      count: count || defaultSearchCount,
+      count: options.count || defaultSearchCount,
+      type: options.type || 'field'
     };
     const queryString = toQueryString(queryParamsObject);
     const url = `${this.config.baseServiceUrl}/api/v2/search${queryString}`;
